notebooks/source/truncated_distributions.ipynb

```diff
@@ -0,0 +1,1383 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #12.        _resolution = jnp.finfo(jnp.result_type(float)).resolution
</pre><p>When we implement a distribution, It is better to avoid invoking JAX ops globally (so modifies like <code>numpyro.enable_x64()</code>  work).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyro-ppl/numpyro/pull/1272/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='20'/>